### ISSUE TITLE
behave emitting PendingDeprecationWarning messages

### ISSUE THREAD

**Issue by adamjcooper** [behave/behave#582]
In one of our builds which runs behave tests (using behave at commit e9c92180efbd57cb2cc740c8a9422d954cf6c1a7), I've noticed lots of lines like this in our build log:

```
c:\users\build-agent\src\behave\behave\runner.py:459: PendingDeprecationWarning: Use 'use_with_user_mode()' instead
```

I think this is due to behave's runner.py itself using the older [user_mode()](https://github.com/behave/behave/blob/master/behave/runner.py#L180-L183) method instead of the newer [use_with_user_mode()](https://github.com/behave/behave/blob/master/behave/runner.py#L176-L178) method:

https://github.com/behave/behave/blob/master/behave/runner.py#L459

**Comment by jenisys** [comment]
FIX INTEGRATED INTO: behave-1.2.6.dev0

**Comment by adamjcooper** [comment]
Thanks @jenisys !
